src/call/pairwise.rs

```diff
@@ -189,7 +189,15 @@ pub fn call<A, B, P, M, R, W, X, F>(
             for (i, event) in events.iter().enumerate() {
                 for (j, pileup) in pileups.iter().enumerate() {
                     let p = if let &Some(ref pileup) = pileup {
-                        Some(pileup.posterior_prob(&event.af_case, &event.af_control))
+                        // if either of the pileups is empty, no valid probability could be
+                        // calculated for it -- and in SingleCellBulkModel's joint_prob() and map()
+                        // this would result in a division by zero and thus a program crash
+                        if  !pileup.case_observations().is_empty() &&
+                            !pileup.control_observations().is_empty() {
+                                Some(pileup.posterior_prob(&event.af_case, &event.af_control))
+                            } else {
+                                None
+                            }
```

Comment:
Actually this is not the case in general. Even if both pileups are empty, you should still get a posterior probability. This probability should simply reflect the prior assumptions. E.g. with flat priors, it should be proportional to the fraction an event takes in the whole event space, with real informative priors, it should be equivalent to the prior expected heterozygosity or mutation rate. Further, if only one pileup is empty the prior for the empty pileup can still help to lead to a more accurate posterior. Imagine a tumor-normal sample pair where I have a good coverage in the tumor, saying that there is a variant at some allele frequency. Even if I have no coverage in the normal, my prior tells me that there is some likelihood of having the same variant in the normal sample. Hence it will nicely adjust my posterior probability.

So my feeling is that if you get a division by zero error in your model, there must be something wrong in there. Maybe you can handle this case in a special way?